tests/storage/storageserver_test.py

```diff
@@ -322,3 +322,17 @@ def test_prepare_connection_without_initiator_name():
     # Unset keys raise KeyError
     with pytest.raises(KeyError):
         con.iface.initiatorName
+
+
+def test_empty_conn_list_on_connect():
+    with pytest.raises(KeyError):
+        storageServer.connect(sd.ISCSI_DOMAIN, [])
+
+
+@pytest.mark.xfail(
+    reason="BZ #2054745, not ready to accept empty connection")
+def test_empty_conn_list_on_disconnect():
+    # If iSCSI target is used by multiple SDs, engine sends empty connection
+    # list and vdsm has to handle empty list on disconnect.
+    # See https://bugzilla.redhat.com/2054745
+    storageServer.disconnect(sd.ISCSI_DOMAIN, [])
```

Comment:
This test can be added for HSM.connectStorageServer.

Reply:
done